source/FreeplayState.hx

```diff
@@ -229,9 +237,9 @@ class FreeplayState extends MusicBeatState
 
 	function changeSelection(change:Int = 0)
 	{
-		#if !switch
-		NGio.logEvent('Fresh');
-		#end
+		// #if !switch
+		// NGio.logEvent('Fresh');
+		// #end
```

Comment:
This should be uncommented

Reply:
Maybe it's just me but having this uncommented will cause the game to crash upon entering freeplay. Will uncomment anyway.

Reply:
it crashes due to newgrounds stuff, should be fixed when #412 is merged

Reply:
Pls pls pls can i request something so my brother want to play the game but its everyday crushes pls pls Pls the crushes pls fix it pls pls pls i begging ypu to fix the crushes pls